lib/ex_force.ex

```diff
@@ -160,6 +161,30 @@ defmodule ExForce do
     do:
       do_get_sobject(client, "sobjects/#{sobject_name}/#{field_name}/#{URI.encode(field_value)}")
 
+  @doc """
+  Makes a request to a SObject based on the value of a specified extneral ID field. Works similarly to get_sobject_by_external_id, but accepts methods other than GET.
```

Comment:
Could you please make following small changes?

1. Keep single line at the first paragraph, as it will be shown in the function list doc
2. Quote the function name with arity, (e.g. ` get_sobject_by_external_id/4`) so that the generated doc has the proper link